packages/components/src/Button/Button.tsx

```diff
@@ -7,6 +7,7 @@ import { Typography } from "../Typography";
 import { Icon, IconNames } from "../Icon";
 
 interface ButtonFoundationProps {
+  readonly ariaLabel?: string;
```

Comment:
Might be good to describe that `ariaLabel` overrides `label` for readers if it is present?

Reply:
@seanhealy addressed